src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,31 @@
+package duke.command;
+
+import duke.exception.FileIoException;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * A command class that encapsulates the action of adding a specific task into the task list.
+ */
+public class AddCommand extends Command {
```

Comment:
It's great that you followed the coding convention for your class names as well!

Reply:
Indeed. Good naming convention used for variable names, class names!